src/components/LoginPage/LoginPageComponent.js

```diff
@@ -1,67 +1,69 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import slide1 from 'components/LoginPage/images/slide-1.png';
-import slide2 from 'components/LoginPage/images/slide-2.png';
-import slide3 from 'components/LoginPage/images/slide-3.png';
-import slide4 from 'components/LoginPage/images/slide-4.png';
+import styled from 'styled-components';
+import Slide1 from 'components/LoginPage/Slide1';
+import Slide2 from 'components/LoginPage/Slide2';
+import Slide3 from 'components/LoginPage/Slide3';
+import Slide4 from 'components/LoginPage/Slide4';
+import Slide5 from 'components/LoginPage/Slide5';
+import iconGoogle from 'components/LoginPage/images/icon-google.svg';
+
+const Main = styled.main`
+  width: 100%;
+  background-color: #131c3e;
+  font-family: Roboto;
+`;
+
+const Article = styled.article`
+  width: 100%;
+  max-width: calc(100vh*1.1);
+  margin: auto;
+`;
+
+const H3 = styled.h3`
+  padding-bottom: 7%;
+  font-size: calc(13px + (24 - 13)*(100vw - 375px)/(1200 - 375));
+  text-align: center;
+  color: #ffffff;
+`;
+
+const Button = styled.a`
```

Comment:
Why `Button` is `<a>` and not `<button>`?

Reply:
Done in commit [04593d4](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/04593d43838bf9d8b3b06a21f9500283428277c6)